code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -2118,9 +2118,9 @@
 		bitesize = 3
 */
 /obj/item/weapon/reagent_containers/food/snacks/mint
-	name = MINT
-	desc = "It is only wafer thin."
-	icon_state = MINT
+	name = "mint"
+	desc = "it is only wafer thin."
```

Comment:
Someone who cares about style as much as you should obviously agree that consistency is important here. It's dumb for this to be the only food that uses a defines for its name and icon.

It's not even your fault, I know it was the REGEX that did it. Don't stand by it mindlessly.